big_tests/tests/carboncopy_helper.erl

```diff
@@ -1,8 +1,80 @@
 -module(carboncopy_helper).
--export([wait_for_carbon_with_body/3]).
+-export([
```

Comment:
In test helper modules we could just export all for simplicity (with `nowarn_export_all`).